logstash-core/lib/logstash/api/commands/node.rb

```diff
@@ -77,6 +80,24 @@ def jvm
         def hot_threads(options={})
           HotThreadsReport.new(self, options)
         end
+
+        private
+        ##
+        # Returns a vertex, decorated with additional metadata if available.
+        # Does not mutate the passed `vertex` object.
+        # @api private
+        # @param vertex [Hash{String=>Object}]
+        # @return [Hash{String=>Object}]
+        def decorate_vertex(vertex)
+          plugin_id = vertex["id"]&.to_s
```

Comment:
what is the `&` for here?

Reply:
Hi @jsvd . Thanks for looking at this! The source of this code originates [from a discussion here](https://github.com/elastic/logstash/pull/10576#discussion_r294942674).